pgx-utils/src/sql_entity_graph/aggregate/entity.rs

```diff
@@ -161,9 +161,9 @@ impl PartialOrd for PgAggregateEntity {
     }
 }
 
-impl Into<SqlGraphEntity> for PgAggregateEntity {
-    fn into(self) -> SqlGraphEntity {
-        SqlGraphEntity::Aggregate(self)
+impl From<PgAggregateEntity> for SqlGraphEntity {
+    fn from(val: PgAggregateEntity) -> Self {
+        SqlGraphEntity::Aggregate(val)
     }
 }
```

Comment:
These are part of a crate that we consider private internal details (anyone who uses them is essentially like a crate enabling rustc attrs... yeah you can do it, but also lol if you think that's stable) and so any change to them would not be a breaking change anyways.

Reply:
Good to know.